openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -13,6 +13,32 @@
 _logger = logging.getLogger(__name__)
 
 
+def login_or_registration_required_at_checkout(cr):
+    """ website_sale_require_login merged into website_sale. Check if the
+    website_sale_require_login module was installed in v15 to set the
+    website.account_on_checkout field as mandatory so that the functionality
+    remains the same, login/registration required for checkout. """
+    # Check if the module is installed and its status is "installed".
+    module_name = "website_sale_require_login"
+    if openupgrade.is_module_installed(cr, module_name):
+        # Add the field 'account_on_checkout' to the 'website' table if it doesn't exist yet.
+        openupgrade.logged_query(
+            cr,
+            """
+            ALTER TABLE website
+            ADD COLUMN IF NOT EXISTS account_on_checkout
```

Comment:
You have to declare the type
```suggestion
            ADD COLUMN IF NOT EXISTS account_on_checkout VARCHAR
```

Reply:
This wasn't fixed...

Reply:
Sorry, my fault for not seeing it. @pilarvargas-tecnativa can you do another PR, please?

Reply:
Yes, I'm doing it right now